strings/junkmail.txt

```diff
@@ -13,3 +13,17 @@ MONTHPEOPLE ARE REAL, AND NO I DON'T MEAN "MOTH" PEOPLE! CALENDAR CRYPTIDS! THE
+Damn, I honestly don't want to deny this any longer. I want to date a syndicate girl. A normal girlfriend is nice and all, but does she know how to arm a nuke? Does she know all the latest equipment in the uplink? Didn't think so. I want my girl to know better than to call office bugs "suspicious devices". She doesn't give a fuck about syndicate revolvers or whatnot. She doesn't use syndicate bombs, cuz they do not give any advantage. Only clothings she would wear is chameleon. She knows she looks hot as fuck without an ID equipped. I know that too. And all that training in the syndicate isn't for nothing. Her CQC is strong as FUCK. If I were jumped by two assistants with spears, she'd shoot the spear-wielding maniacs, and whisper: "Alpha, Alpha, Delta." into my ear, as she holds me tight to assure me we aren't under fire anymore. Not to mention that we both love traitor gear. Our evenings would consist of heated arguments like whether The Sleeping Carp or the Double-Bladed Energy Sword rules the Syndicate world. All in all, I really fucking want to date and marry a syndicate girl.
+I just got booted from EVERY syndicate allied servers (including black-site ones) for "not escalating properly". WHAT THE FUCK DOES THAT EVEN MEAN. Please help...
+Contrived, mediocre, milquetoast, amateurish, infantile, cliche-and-gonorrhea-ridden pean to conformism, affront to humanity, war crime, should *litteraly* be tried for war crimes, resolutely shit, lacking in imagination, uninformed reimagining of, limp-wristed, premature, ill-informed attempt at, talentless fuckfest, recidivistic shitpeddler, pedantic, listless, savagely boring, just one repulsive laugh after another.
+Evolutionists say that dinosaur 🦕 evolve from pig. This is stupid pig is a mammals 2. Evolurionisr say that whale are mammal and there troat is snall. This id bulldhit😒 according to the Bible❤ whale is a fish all animals in the water are fish and whale can eat any thing bigger them selves 3. Evolutionist say that monkey through shit💩 to evolve to be a human. Wow these evolutionist is disgusting 4. Carbon dating dont exist. Uranium is all made up pls wake up.
```

Comment:
these emojii render in game?

Reply:
Line 4 had emojis and it did render in-game pre-paper refactor, I assume it still does now.

Reply:
![image](https://user-images.githubusercontent.com/53777086/193187698-21d4a4be-298b-4e8f-806d-b4449af35937.png)